liquidator/OneInchExchange.js

```diff
@@ -24,20 +22,53 @@ class OneInchExchange {
     this.oneSplitContract = new web3.eth.Contract(OneSplit.abi, this.oneSplitAddress);
 
     this.toBN = web3.utils.toBN;
+
+    // 1 Split config
+    this.oneInchFlags = 0; // Enables all exchanges
+
+    // Number of pieces source volume could be splitted
```

Comment:
nit: our formatting style for the repo dictates 120 char line length. I think you could wrap this comment to far few lines if you made the full use of the space available to you.